### DISCUSSION TITLE
Is there a "skip song" AzuraCast command?

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7100] [Q&A] [answered]
Hi.

Is there a "skip song" AzuraCast command of some kind? I've checked the API but there is no such option...

Thank you.

**Comment by BusterNeece** [comment, marked as answer]
@BornKillaz There definitely is: it's via the "Skip Song" button on the station profile or by making a `POST` request to `/api/station/(id)/backend/skip`.

**Reply by BornKillaz** [reply]
Oh, thank you so much.

Unfortunately that command (backend/skip) is not documented here https://www.azuracast.com/api/

**Reply by BusterNeece** [reply]
@BornKillaz It is actually! It's just kind of hard to see from the main directory of actions: https://www.azuracast.com/api/#/Stations%3A%20Service%20Control/doBackendServiceAction

All of the "backend service" actions (that would apply to Liquidsoap) are all under a single API endpoint. One of the actions you can do is "skip", which informs the backend to skip a track.
